automated install/basic-install.sh

```diff
@@ -2796,6 +2796,11 @@ main() {
 
     stop_service pihole-FTL &> /dev/null
 
+    if [ -d /var/log/pihole/ ]; then
```

Comment:
```suggestion
    if [ ! -d /var/log/pihole/ ]; then
```
I'm sorry 😄.

Reply:
No worries, thanks for the catch!

Reply:
But the PR didn't get the catch (at line 2799)?  Or am I missing something?

Reply:
There's another PR that has the change.